.github/workflows/linter.yaml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - incubation
+      - rhoai
```

Comment:
not really a doc

Reply:
removed, put in https://github.com/opendatahub-io/opendatahub-operator/pull/1593 for change